opencsp/common/lib/csp/FacetEnsemble.py

```diff
@@ -219,6 +205,9 @@ def set_facet_positions(self, positions: Pxyz):
             facet._self_to_parent_transform = TransformXYZ.from_V(pos)
 
     def set_facet_canting(self, canting_rotations: list[Rotation]):
+        """Sets facet canting relative to ensemble.
+        NOTE: Will remove previously set facet positionals
+        """
```

Comment:
```suggestion
    """
    Set the canting rotations of the facets relative to the ensemble.

    This function updates the canting rotations of the facets in the 
    ensemble. It will remove any previously set facet positional 
    transformations.

    Parameters
    ----------
    canting_rotations : list[Rotation]
        A list of rotation objects to set for each facet. The length 
        of this list must match the number of facets in the ensemble.

    Raises
    ------
    ValueError
        If the length of `canting_rotations` does not match the number 
        of facets in the ensemble.

    Notes
    -----
    This method modifies the internal transformation of each facet 
    based on the provided canting rotations and their corresponding 
    positions.
    """
    # "ChatGPT 4o-mini" assisted with generating this docstring.
```

Reply:
@e10harvey, thanks for looking at this. I updated the documentation as per your suggestions. 
And to answer your other question, it doesn't look like OpticOrientationAbstract is explicitly tested anywhere. However, it may be indirectly tested in test_MirrorPoint and test_MirrorParametric.